index.html

```diff
@@ -40,6 +41,160 @@ <h1>I'm Hossain.<br>Glad to see you! </h1>
         <li><a href="#" title="Angellist"><img src="img/angellist.svg" alt="angellist logo"></a></li>
       </ul>
     </section>
+     <!-- Work -->
+    <section id="work">
+      <div class="work-first flex-column">
+        <div class="work-description flex-column">
+          <div> <img src="img/Portfolio-1.png" alt=""></div>
+          <div>
+            <div class="specification">
+              <h2>Hossain</h2>
+              <div class="flex">
+                <h4>CANOPY</h4>
+                <!-- <p>&#x2022;</p>
+                <P> Full Stack Dev</P>
+                <p>&#x2022;</p>
+                <p>2023</p> -->
+                <ul class="flex">
+                  <li>Full Stack Dev</li>
+                  <li>2023</li>
+                </ul>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="work-description">
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        </div>
+        <div class="work-description flex">
+          <button class="language-btn"> <p>html</p></button>
+          <button class="language-btn"><p>css</p></button>
+          <button class="language-btn"><p>Javascript</p></button>
+        </div>
+        <button class="see-project-btn">See Project</button>
+      </div>
+      <div class="work-first flex-column">
+        <div class="work-description flex-column">
+          <div> <img src="img/Portfolio-2.png" alt=""></div>
```

Comment:
- [ ] Following the best practices, all image tags must have a valid alt attribute, which makes the images easy to identify by a screen reader, I'll suggest you go through your project list and add the correct text in the image for all attributes. :+1: :smile: